front/src/index.ts

```diff
@@ -10,10 +10,11 @@ import {FourOFourScene} from "./Phaser/Reconnecting/FourOFourScene";
 import WebGLRenderer = Phaser.Renderer.WebGL.WebGLRenderer;
 import {OutlinePipeline} from "./Phaser/Shaders/OutlinePipeline";
 import {CustomizeScene} from "./Phaser/Login/CustomizeScene";
-import {HtmlUtils} from "./WebRtc/HtmlUtils";
 import {CoWebsiteManager} from "./WebRtc/CoWebsiteManager";
+import {redirectIfToken} from "./register";
 
 //CoWebsiteManager.loadCoWebsite('https://thecodingmachine.com');
+redirectIfToken();
```

Comment:
If there URL matches "register", the application will perform an "Axios" query.
But while this query is being performed, the application will continue loading (which might lead to the screen flickering because Phaser 3 might have the time to load the "LoginScreen" before we are redirected.

Maybe redirectIfToken might return "true" or "false" based on the URL (and if it returns true, we don't even try to initialize phaser and simply wait for the redirection?)